test/replication-luatest/gh_7486_deprecate_replication_synchro_timeout_test.lua

```diff
@@ -0,0 +1,42 @@
+local t = require('luatest')
+local server = require('luatest.server')
+
+local g = t.group('deprecate_replication_synchro_timeout')
+--
+-- gh-7486: deprecate `replication_synchro_timeout`.
+--
+
+g.after_each(function(cg)
+    cg.server:drop()
+end)
+
+g.test_option_available_if_old = function(cg)
+    cg.server = server:new({alias = 'old_behavior'})
```

Comment:
We'll have to rewrite the test, when default compat will be changed. Better to set it explicitly to old now

Reply:
Fixed.